src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -1100,7 +1100,9 @@ private static XNode[] RewriteMany(ISymbol symbol, Compilation compilation, XNod
                     if (methodSymbol.MethodKind == MethodKind.Constructor || methodSymbol.MethodKind == MethodKind.StaticConstructor)
                     {
                         var baseType = memberSymbol.ContainingType.BaseType;
+#nullable disable // Can 'baseType' be null here? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
Yes, if somebody put an inheritdoc on constructor of System.Object itself?

Reply:
Or on an interface member?

Reply:
This is the form for lines I am not dealing with as part of this pull request. The comment is just so the reader can see which part of the line resulted in a warning.